test/runtests.sh

```diff
@@ -1,28 +1,8 @@
-#-------------------------------------------------------------------------------------------------------
 # Copyright (C) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 #-------------------------------------------------------------------------------------------------------
 #
-# Eventual test running harness on *nix for ChakraCore
-# Today, it's simply there to make sure hello.js doesn't regress
-#
+# REMOVE THIS AFTER MERGING runtests.py in!
```

Comment:
I don't think there's any need for this comment. We can make this decision later.